features-json/jpegxl.json

```
@@ -240,7 +240,7 @@
       "89":"n",
       "90":"n",
       "91":"n",
```

```suggestion
      "91":"n d #1",
```
This will also be available in Chrome 91, see [this comment](https://bugs.chromium.org/p/chromium/issues/detail?id=1178058#c13).